Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -2372,26 +2372,36 @@ def prettify_get_url_filter(url_filter: dict):
     if 'override' in url_filter:
         override_category_list = url_filter['override']['member']
 
+    if not isinstance(alert_category_list, list):
+        alert_category_list = [alert_category_list]
```

Comment:
why not arg_to_list()? (and for the others)

Reply:
correct, will fix.